<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPump</class>
 <widget class="QDialog" name="DialogPump">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="_label_1">
     <property name="text">
      <string>Infuse rate</string>
     </property>
     <property name="buddy">
      <cstring>spin_rate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDoubleSpinBox" name="spin_rate">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>1999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="combo_units">
     <item>
      <property name="text">
       <string>ml/min</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>µl/min</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ml/hr</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>µl/hr</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="_label_2">
     <property name="text">
      <string>Diameter</string>
     </property>
     <property name="buddy">
      <cstring>spin_diameter</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDoubleSpinBox" name="spin_diameter">
     <property name="suffix">
      <string> mm</string>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>1999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="_label_3">
     <property name="text">
      <string>Target infusion</string>
     </property>
     <property name="buddy">
      <cstring>spin_target</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDoubleSpinBox" name="spin_target">
     <property name="suffix">
      <string> ml</string>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.005000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>219</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogPump</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogPump</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
